integration/GET_KERNEL_QEMU

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# This script is intended to run, locally, the tests we run at circleci,
+# precisely as they are run there.
+#
+# to do so, it:
+# o creates a directory to store local artifacts retrieved from docker
+#   see TMP= below
+# o runs a the standard test container to retrieve a the qemu, kernel, and bios image
+# o runs go test with a default set of tests (./...)
+#
+# NOTE: if you want more complex behavior, don't make this script more
+# complex. Convert it to Go. Complex shell scripts suck.
+
+# These docker artifacts should not persist. Place them in tmp.
+# tmp is in .gitignore
+# I would prefer /tmp/$$.
+# Docker really doesn't like this for some reason, even when
+# I map it to /out inside the container.
+# TMP=/tmp/$$
+TMP=`pwd`/tmp
```

Comment:
I was unable to get anything save a local tmp to work. Which may be ok, in this case, since it's already in a per-user place and saves littering /tmp with unchanging qemu and kernel for each time you run it. In the end, I think this is preferable.